tools/third_party/pywebsocket3/mod_pywebsocket/stream.py

```diff
@@ -875,7 +875,7 @@ def close_connection(self,
             wait_response: True when caller want to wait the response.
         Raises:
             BadOperationException: when reason is specified with code None
-            or reason is not an instance of both str and unicode.
+                or reason is not an instance of both str and unicode.
```

Comment:
We can't modify this in place.

Could you add an exception for this for now? I'll send a quick PR to `pywebsocket3` and roll it in.

Reply:
I have a [PR](https://github.com/GoogleChromeLabs/pywebsocket3/pull/16) that has been merged in `pywebsocket3`